pwa/src/components/translations/tableNamesTable.tsx

```diff
@@ -1,100 +1,61 @@
 import * as React from "react";
-import {
-  Table,
-  Card,
-  Spinner,
-  Modal
-} from "@conductionnl/nl-design-system/lib";
 import { Link } from "gatsby";
+import { Table, Card, Spinner } from "@conductionnl/nl-design-system/lib";
 import APIService from "../../apiService/apiService";
 import APIContext from "../../apiService/apiContext";
-import { AlertContext } from "../../context/alertContext";
-import { HeaderContext } from "../../context/headerContext";
 
 export default function TableNamesTable() {
-  const [documentation, setDocumentation] = React.useState<string>(null);
   const [tableNames, setTableNames] = React.useState<Array<any>>(null);
   const [showSpinner, setShowSpinner] = React.useState<boolean>(false);
   const API: APIService = React.useContext(APIContext);
-  const [_, setAlert] = React.useContext(AlertContext);
-  const [__, setHeader] = React.useContext(HeaderContext);
-
-  React.useEffect(() => {
-    setHeader({ title: "Table names", subText: "An overview of your table names" });
-  }, [setHeader]);
-
+  
   React.useEffect(() => {
-    handleSetDocumentation();
-  });
-
-  React.useEffect(() => {
-    handleSetTableNames()
+    getTableNames()
   }, [API]);
 
-  const handleSetTableNames = () => {
+  const getTableNames = () => {
     setShowSpinner(true);
-    API.Translation.getAllTableNames()
-      .then((res) => {
-        const convertedArray = res.data["results"].map((value) => ({
-          name: value
-        }));
-        setTableNames(convertedArray);
-      })
-      .catch((err) => {
-        setAlert({ message: err, type: "danger" });
-        throw new Error("GET table names error: " + err);
+    API.Translation.getTableNames()
+      .then((res) => { 
+        const names = res.data.results.map((name) => { 
+          return { name: name }
+        });
```

Comment:
When using a `.map()` function, the shorthand way of returning the items is by **not adding** the `{}`, like so:

```
const names = res.data.results.map((name) => { name: name } );
```
_or_
```
const names = res.data.results.map((name) => { ...{ name } });
```

**Note: ** I can imagine this being confusing. The `{}` here are not the brackets of opening a function but the brackets of opening the object we're returning 😛